.github/workflows/validate_bids-examples.yml

```diff
@@ -0,0 +1,144 @@
+name: validate_datasets
+
+on:
+  push:
+    branches: ['master']
+  pull_request:
+    branches: ['**']
+#  create:
+#    branches: [master]
+#    tags: ['**']
+#  schedule:
+#    - cron: "0 4 * * 1"
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [ubuntu-latest] # , macos-latest, windows-latest]
+        bids-validator: [master-deno]
+        python-version: ["3.11"]
+
+    runs-on: ${{ matrix.platform }}
+
+    env:
+      TZ: Europe/Berlin
+      FORCE_COLOR: 1
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: "Apply patches for BIDS-2.0"
+      run: |
+        set -o pipefail
+        tools/bids-2.0/apply_all 2>&1 | tee /tmp/patch.log
```

Comment:
after RFing to have this patching to be done here instead of statically in the commit  testing started to fail

```
=========================== short test summary info ============================
FAILED tests/test_validator.py::test_bids_datasets[eeg_cbm] - AssertionError: assert 2 == 0
 +  where 2 = len(['/tmp/tmp71v8bn_s/eeg_cbm/subjects.tsv', '/tmp/tmp71v8bn_s/eeg_cbm/subjects.json'])
FAILED tests/test_validator.py::test_bids_datasets[micr_SEMzarr] - AssertionError: assert 2 == 0
 +  where 2 = len(['/tmp/tmp71v8bn_s/micr_SEMzarr/subjects.tsv', '/tmp/tmp71v8bn_s/micr_SEMzarr/subjects.json'])
FAILED tests/test_validator.py::test_bids_datasets[micr_SPIM] - AssertionError: assert 2 == 0
 +  where 2 = len(['/tmp/tmp71v8bn_s/micr_SPIM/subjects.tsv', '/tmp/tmp71v8bn_s/micr_SPIM/subjects.json'])
FAILED tests/test_validator.py::test_bids_datasets[pet003] - AssertionError: assert 2 == 0
 +  where 2 = len(['/tmp/tmp71v8bn_s/pet003/subjects.tsv', '/tmp/tmp71v8bn_s/pet003/subjects.json'])
FAILED tests/test_validator.py::test_bids_datasets[ds000248] - AssertionError: assert 2 == 0
 +  where 2 = len(['/tmp/tmp71v8bn_s/ds000248/subjects.tsv', '/tmp/tmp71v8bn_s/ds000248/subjects.json'])
============= 5 failed, 62 passed, 1 skipped, 7 warnings in 15.67s =============
```
so as if testing against not patched schema... can't figure it out why :-/